modules/nw-operator-vsphere-multiple-nic.adoc

```diff
@@ -0,0 +1,86 @@
+====
+
+.Example error message because of the {vmw-short} CCM selecting an incompatible NIC
+[source, text]
+----
+ERROR Bootstrap failed to complete: timed out waiting for the condition
+ERROR Failed to wait for bootstrapping to complete. This error usually happens when there is a problem with control plane hosts that prevents the control plane operators from creating the control plane.
+----
+
+.Prerequisites
+
+* You created Kubernetes manifest files for your {product-title} cluster.
+
+.Procedure
+
+. Enter the following command in the {oc-first} tool to view detailed information, especially `Internal-IP` or `External-IP information, for each node that exists in your cluster. You will need this information in a the `nodeNetworking:` section of the `nodeNetworking` object.
++
+[source,terminal]
+----
+$ oc get nodes -o wide
+----
+
+. From the directory where you store your {product-title} cluster manifest files, open the `manifests/cluster-infrastructure-02-config.yml` manifest file.
+
+. Add a `nodeNetworking` object to the file and specify internal and external network subnet CIDR implementations for the object.
++
+.Example of a configured `cluster-infrastructure-02-config.yml` manifest file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+  name: cluster
+spec:
+  cloudConfig:
+    key: config
+    name: cloud-provider-config
+  platformSpec:
+    type: VSphere
+    vsphere:
+      failureDomains:
+      - name: generated-failure-domain
+      ...
+       nodeNetworking:
+         external: <1>
+           networkSubnetCidr:
+           - <machine_network_cidr_ipv4>
+           - <machine_network_cidr_ipv6>
+         internal: <2>
+           networkSubnetCidr:
+           - <machine_network_cidr_ipv4>
+           - <machine_network_cidr_ipv6>
+      vcenters:
+      - datacenters:
+        - sample-dc
+        port: 443
+        server: vcenter.example.com
+status:
+  ...
+  platformStatus:
+    type: VSphere
+# ...
+----
+<1> The IP address of a node that exists outside to your cluster. The CIDR must match the CIDR that you specified in the `networking.machineNetwork` section of your `install-config.yaml` file.
+<2> The IP address of a node that exists inside your cluster. The CIDR must match the CIDR that you specified in the `networking.machineNetwork` section of your `install-config.yaml` file.
++
+[NOTE]
+====
+If you do not set values for the `internal.networkSubnetCIDR` or `external.networkSubnetCIDR` parameters, the NIC related to the network must only have a single IP address as this address is set as the default value for each of these parameters.
```

Comment:
This is copy-pasted out of context from the API docs and I think it's misleading since it applies to setting these values based on network names, not CIDRs. In the example we're documenting the CIDR parameters would never not be set. Unless we're going to include an example using network names this note should not be here.

Reply:
Removed the note:

```
[NOTE]
====
If you do not set values for the `internal.networkSubnetCIDR` or `external.networkSubnetCIDR` parameters, the NIC related to the network must only have a single IP address as this address is set as the default value for each of these parameters.
====
```